op-node/rollup/derive/engine_consolidate.go

```diff
@@ -37,3 +43,33 @@ func AttributesMatchBlock(attrs *eth.PayloadAttributes, parentHash common.Hash,
 	}
 	return nil
 }
+
+// logL1InfoTxns reports the values from the L1 info tx when they differ to aid
+// debugging. This check is the one that has been most frequently triggered.
+func logL1InfoTxns(l log.Logger, l2Number, l2Timestamp uint64, safeTx, unsafeTx hexutil.Bytes) {
+	// First decode into *types.Transaction to get the tx data.
+	var safeTxValue, unsafeTxValue types.Transaction
+	errSafe := (&safeTxValue).UnmarshalBinary(safeTx)
+	errUnsafe := (&unsafeTxValue).UnmarshalBinary(unsafeTx)
+	if errSafe != nil || errUnsafe != nil {
+		l.Error("failed to umarshal tx", "errSafe", errSafe, "errUnsafe", errUnsafe)
```

Comment:
Needs a return if the transactions failed to decode.

Reply:
nice catch